parser/src/parser.cpp

```diff
@@ -82,7 +82,10 @@ po::options_description commandLineArguments()
      "further actions modifying the state of the database.")
     ("incremental-threshold", po::value<int>()->default_value(10),
       "This is a threshold percentage. If the total ratio of changed files "
-      "is greater than this value, full parse is forced instead of incremental parsing.");
+      "is greater than this value, full parse is forced instead of incremental parsing.")
+    ("log-target", po::value<std::string>(),
```

Comment:
I think this would be better placed after `loglevel`. The naming convention should also be unified, either `log-level` or `logtarget`.

Reply:
Changed it to "logtarget".